Source/Core/Core/HW/DVDInterface.h

```diff
@@ -106,20 +106,16 @@ void DoState(PointerWrap& p);
 
 void RegisterMMIO(MMIO::Mapping* mmio, u32 base);
 
-// Direct disc access
+// Disc access (don't call GetVolume unless you know that IsDiscInside() == true)
```

Comment:
I didn't look at that function (so maybe it's already available) but could we potentially assert that this is true?  Or maybe add panic-handlers?